src/psyclone/dynamo0p3_builtins.py

```diff
@@ -553,10 +579,11 @@ def gen_code(self, parent):
 
 
 class DynIncXMinusYKern(DynBuiltIn):
-    ''' Subtract the second field from the first field and return it '''
+    ''' Subtract the second real-valued field from the first field
+    and return it '''
```

Comment:
Full stop please. Please could you put the blank line that comes after the docstring inside the docstring.

Reply:
Yep, I have done this for all built-in classes.